### DISCUSSION TITLE
Unique Option for Fields

### DISCUSSION THREAD

**Discussion by dasmeet** [directus/directus#3049] [Feature Requests]
Like v8, it would be nice to have an option to make a field unique in the table. Unless it's there and I am somehow missing it :)

![image](https://user-images.githubusercontent.com/46486687/94787305-c7dabb80-03ef-11eb-9381-9a92b899c629.png)

**Comment by rijkvanzanten** [comment]
Unique field constraints have since been implemented 🙂
